editors/code/src/commands.ts

```diff
@@ -127,8 +127,8 @@ export function joinLines(ctx: Ctx): Cmd {
             ranges: editor.selections.map((it) => client.code2ProtocolConverter.asRange(it)),
             textDocument: ctx.client.code2ProtocolConverter.asTextDocumentIdentifier(editor.document),
         });
-        await editor.edit((builder) => {
-            client.protocol2CodeConverter.asTextEdits(items).forEach((edit: any) => {
+        await editor.edit(async (builder) => {
```

Comment:
Can we get a cancellation token into the command handlers? There must be a way, but I missed it.